x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/fields_browser.cy.ts

```diff
@@ -49,26 +47,15 @@ const defaultHeaders = [
   { id: 'user.name' },
 ];
 
-// Flaky in serverless tests
-// FLAKY: https://github.com/elastic/kibana/issues/169363
-describe.skip('Fields Browser', { tags: ['@ess', '@serverless'] }, () => {
-  context('Fields Browser rendering', () => {
+describe('Fields Browser', { tags: ['@ess', '@serverless'] }, () => {
+  describe('Fields Browser rendering', () => {
     beforeEach(() => {
       login();
       visitWithTimeRange(hostsUrl('allHosts'));
       openTimelineUsingToggle();
-      populateTimeline();
       openTimelineFieldsBrowser();
     });
 
-    it('displays all categories (by default)', () => {
-      cy.get(FIELDS_BROWSER_SELECTED_CATEGORIES_BADGES).should('be.empty');
-    });
-
-    it('displays "view all" option by default', () => {
-      cy.get(FIELDS_BROWSER_VIEW_BUTTON).should('contain.text', 'View: all');
-    });
-
     it('displays the expected count of categories that match the filter input', () => {
```

Comment:
what do you think about merging some of the tests below?

I can see the first (displays the expected count of categories that match the filter input) and second (displays a search results label with the expected count of fields matching the filter input) together. They have the same input and only check a small UI element each...

I can also see the last 3 (or 2) in this describe merged together, under a title that would say something like `should correctly create and search category` or something more descriptive. A few `cy.log` in the test would clarify what exactly happens in each step.

The whole test suite would run much faster

Reply:
Yep, nice catch. I'll merge them.